<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>594</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>594</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SensPrecOptimizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="fldInput">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="fldOutput">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOutput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInput">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutput">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>47</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Path</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="boxDataConfigs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>573</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Configure Data</string>
    </property>
    <widget class="QLineEdit" name="fldTotalRelevancy">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRelevancy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Total Number of Relevant Documents:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cBoxDefaultConfig">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblRelevancyConfig">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalRelevancy">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Custom</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="boxAlgoConfigs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>573</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Configure Algorithm</string>
    </property>
    <widget class="QLabel" name="lblCombStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Combinations Iterations - The Starting Point </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="fldCombStart">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>75</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fldCombSmaller">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCombSmaller">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Combinations Iterations - The Number of Additional Smaller Neighbours </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPercent_1">
     <property name="geometry">
      <rect>
       <x>551</x>
       <y>20</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="gBoxFilterOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Runtime Filtering of Output</string>
     </property>
     <widget class="QLabel" name="lblSens">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Acceptable Minumum Sensitivity:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="cBoxDefaultMinSens">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblPercent_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="fldMinSens">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMinSens">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Custom</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblDefaultMinSens">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="txtLog">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="boxOutputConfigs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Configure Output Filter</string>
    </property>
    <widget class="QLabel" name="lblFilterReport">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Short Report by Minimising Search Strategies' Length</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="cBoxFilterReport">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblPercent_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblFilterSens">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Report within Sensitivity Range of </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="fldFilterSens">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbort">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSubHelp"/>
    <addaction name="actionSubAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>SenPrecOptim</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Alt+H</string>
   </property>
  </action>
  <action name="actionSubHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSubAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>fldInput</tabstop>
  <tabstop>btnInput</tabstop>
  <tabstop>fldOutput</tabstop>
  <tabstop>btnOutput</tabstop>
  <tabstop>fldMinSens</tabstop>
  <tabstop>btnRun</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
